src/main/java/uk/gov/hmcts/reform/sendletter/tasks/UploadLettersTask.java

```diff
@@ -51,73 +52,78 @@ public UploadLettersTask(
     @SchedulerLock(name = TASK_NAME)
     @Scheduled(fixedDelayString = "${tasks.upload-letters.interval-ms}")
     public void run() {
-        if (!availabilityChecker.isFtpAvailable(now(ZoneId.of(EUROPE_LONDON)).toLocalTime())) {
-            logger.info("Not processing '{}' task due to FTP downtime window", TASK_NAME);
-            return;
-        }
-
         logger.info("Started '{}' task", TASK_NAME);
 
-        List<Letter> lettersToUpload = repo.findFirst3ByStatus(LetterStatus.Created);
-        int counter = 0;
-
-        if (!lettersToUpload.isEmpty()) {
-            counter = ftp.runWith(sftpClient -> uploadLetters(lettersToUpload, sftpClient));
+        if (!availabilityChecker.isFtpAvailable(now(ZoneId.of(EUROPE_LONDON)).toLocalTime())) {
+            logger.info("Not processing '{}' task due to FTP downtime window", TASK_NAME);
+        } else {
```

Comment:
now with this move makes it even more sensible to move `started this task` to aop component which we already have specific for schedules. just a blarb, not take care right now